### DISCUSSION TITLE
Domain not accessible when entering it in with "www."

### DISCUSSION THREAD

**Discussion by bitsmyth** [vercel/vercel#4252] [General] [answered]
I have successfully transfered my domain to vercel now. But my problem is that the domain is not accessible when entering it with `www.`mysite.com.
I am using the newest release of nextjs and I haven´t set up any custom config.
What configuration am I missing?

**Comment by bitsmyth** [comment, marked as answer]
I have solved this pretty quickly actually, by simply adding the domain twice to my project. 
One domain is using `www` and the other isn´t.

**Reply by wiiiimm** [reply]
@NikoMontana for SEO, you should also set one domain to redirect to another. This can be done under your project's production domain settings.

**Reply by bitsmyth** [reply]
@williamli What direction would you advise? 
Redirecting from `www` to `mysite.com` or the other way around? What are your thoughts about this as you mentioned it affects SEO?

**Reply by adrianlawrence** [reply]
Its a matter of personal preference - but from an SEO point of view you want all links to point to just one version so the benefit of the backlinks is not diluted across the different versions.

**Comment by adrianlawrence** [comment]
Yes those are actually different sub domains and they need their own A records setup in order to correctly handle web queries.

**Reply by ihorjs** [reply]
@williamli 
For SEO, I want one domain to redirect to another.  I want to redirect mysite.com to www.mysite.com.
I followed this guide and mysite.com isn't redirecting.
https://vercel.com/docs/v2/custom-domains#redirecting-domains

Am I missing sth?
